---
 intl/locale.alias |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: glibc-2.28/intl/locale.alias
===================================================================
--- glibc-2.28.orig/intl/locale.alias
+++ glibc-2.28/intl/locale.alias
@@ -43,8 +43,8 @@ danish          da_DK.ISO-8859-1
 dansk		da_DK.ISO-8859-1
 deutsch		de_DE.ISO-8859-1
 dutch		nl_NL.ISO-8859-1
-eesti		et_EE.ISO-8859-1
-estonian	et_EE.ISO-8859-1
+eesti		et_EE.ISO-8859-15
+estonian	et_EE.ISO-8859-15
 finnish         fi_FI.ISO-8859-1
 french		fr_FR.ISO-8859-1
 galego		gl_ES.ISO-8859-1
